internal/commands/result.go

```diff
@@ -1263,27 +1277,35 @@ func parseGlSastVulnerability(result *wrappers.ScanResult, glSast *wrappers.GlSa
 		Message:     message,
 		Description: result.Description,
 		CVE:         ID,
-		Severity:    result.Severity,
-		Confidence:  result.Severity,
+		Severity:    cases.Title(language.English).String(result.Severity),
+		Confidence:  cases.Title(language.English).String(result.Severity),
 		Solution:    "",
+
 		Scanner: wrappers.GlScanner{
 			ID:   category,
 			Name: category,
 		},
-		Links: nil,
-		Tracking: wrappers.Tracking{Items: wrappers.Item{
-			Signatures: wrappers.Signature{
-				Algorithm: result.Type + "-Algorithm ",
-				Value:     "NA"},
-			File:      fileName,
-			EndLine:   endLine,
-			StartLine: startLine},
+		Identifiers: []wrappers.Identifier{
+			{
+				Type:  "similarityId",
+				Name:  "Similarity Id ",
+				URL:   wrappers.AnalyzerURL,
+				Value: result.ID,
+			},
 		},
-		Flags: wrappers.Flag{
-			Type:        "",
-			Origin:      result.Type,
-			Description: result.Description,
+		Links: make([]string, 0),
+		Tracking: wrappers.Tracking{
+			Type: "source",
+			Items: []wrappers.Item{
+				{
+					Signatures: []wrappers.Signature{{Algorithm: result.Type + "-Algorithm ", Value: "NA"}},
```

Comment:
Maybe we could use a format string with fmt.Sprintf to avoid the concat

Reply:
ya it makes, i will ask Margarita to change in the next PR